config/v1/types_ingress.go

```diff
@@ -54,6 +54,24 @@ type IngressSpec struct {
 	// configurable routes.
 	// +optional
 	ComponentRoutes []ComponentRouteSpec `json:"componentRoutes,omitempty"`
+
+	// requiredHSTSPolicies specifies HSTS policies that are required to be set on
+	// newly created routes matching criteria that are specified in the policy.
+	// - If there are no RequiredHSTSPolicies, any HSTS Policy annotation on the route is valid.
+	// - If the route specifies a HSTS Policy annotation that matches one of the RequiredHSTSPolicies,
```

Comment:
I don't think this should be true.  I think the matching logic should follow https://github.com/openshift/api/pull/958/files#r661739491 .  If it does, then its entirely possible to have two RequiredHSTSPolicies match, one accept, one reject.  And if the one rejecting is first, the route should be rejected.

Reply:
Agreed.